skycoin/skywire
# Add the killswitch and secure options to the manager

Did you run `make format && make check`?
The go code was not changed. `npm run lint` and `npm run build` were used.

Fixes #609 

 Changes:	
- A checkbox for activating/deactivating the killswitch option was added to the modal window used for configuring the vpn-client app. Additional changes were added to the modal window for being able to add the new configuration option independently from the server selection options.

- A checkbox for activating/deactivating the secure option was added to the modal window used for configuring the vpn-server app.

Note: changes mentioned in https://github.com/skycoin/skywire/issues/609#issuecomment-732908997 are needed before finishing the PR.

How to test this PR:
Open the modal windows used for configuring the vpn-client and vpn-server apps and check:

- If the changes are correctly saved in the visor.
- If after opening the modal windows the UI shows the options checked or unchecked as appropriate, depending on the current configuration saved in the visor.
